src/main/java/org/dice_research/opal/civet/metrics/Description.java

```diff
@@ -0,0 +1,87 @@
+package org.dice_research.opal.civet.metrics;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.rdf.model.SimpleSelector;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.vocabulary.DCAT;
+import org.apache.jena.vocabulary.DCTerms;
+import org.apache.jena.vocabulary.RDF;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dice_research.opal.civet.Metric;
+import org.dice_research.opal.common.vocabulary.Opal;
+
+/**
+ * The Description metric awards stars based on the the different factors in the
+ * dataset
+ * 
+ * @author Aamir Mohammed
+ */
+public class Description implements Metric {
+
+	public static int compareDescWithTitle(String desc, String title, int score) {
+		if (desc.length() >= title.length()) {
```

Comment:
In this case, 
if desc.length() = title.length(), then you will return a 7 Star rating. 

I suggest you change this approach from just comparing strings length. 

You can look into Levenshtein Distance, and give star rating based on the dissimilarity of two strings in desc and title.
https://www.baeldung.com/java-levenshtein-distance